app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -783,15 +788,17 @@ private void closeTab(int index) {
           updateTabSwitcherIcon();
         })
         .show();
-    openDefaultTab();
-    updateTabSwitcherIcon();
+    if (webViewList.size() == 0) {
+      openDefaultTab();
+    } else {
+      updateTabSwitcherIcon();
+    }
   }
 
   private void openDefaultTab() {
     new Handler().postDelayed(() -> {
-      if (webViewList.size() == 0) {
         newTab(HOME_URL);
```

Comment:
Use 2 space indents instead of 4.

Reply:
Sure. I made the changes. I will follow the coding guidelines on all future commits.